### PR TITLE
Fix some Clang warnings

### PR DESCRIPTION
**Pull request by tekknolagi**

### CODE REVIEW

File: src/mc.c [hunk 1]
```diff
@@ -2565,6 +2565,7 @@ int od_state_get_predictor(od_state *state,
   int mvb_sz;
   int ncns;
   int ci;
+  (void)ref; /* fix unused parameter warning */
```
Comment (smarter) [inline src/mc.c]: We have `OD_UNUSED` for that.

Comment (tekknolagi) [inline src/mc.c]: Just added this.

**Comment by smarter** [comment]
I suggest squashing these two commits into one commit and adding "Fix #134" to the commit message, so that #134 is automatically closed when this commit gets merged.

**Comment by tekknolagi** [comment]
Can do.

**Comment by tekknolagi** [comment]
Done!

**Comment by smarter** [comment]
I just remembered that we have a patch at https://review.xiph.org/988/ that uses the `ref` parameter in `od_state_get_predictor`, I'll merge your patch without the `OD_UNUSED` line to avoid forgetting to remove it when https://review.xiph.org/988/ is merged.

**Comment by tekknolagi** [comment]
So should I go ahead and remove `OD_UNUSED`?

**Comment by smarter** [comment]
it's fine, I did it myself and merged your commit: https://git.xiph.org/?p=daala.git;a=commit;h=5f131b5b353c9df4ea4264905a96758c8dfe59db . Thank you for your contribution!

**Comment by tekknolagi** [comment]
Excellent! Glad I could contribute.

**Pull request closed without merging**